docs/database/db_schema-pa.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Copyright 2015 ISRG.  All rights reserved
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+--
+-- This file defines the table schema, foreign keys and indicies of the
+-- Policy Authority database, which is logically separate from that which
+-- is utilized by the Storage Authority and administrator tools.
+--
+
+CREATE TABLE `ruleList` (
+  `rule` varchar(255) NOT NULL,
+  `type` char(11) NOT NULL
```

Comment:
This is a situation where one could argue for the ENUM type.

The ENUM type, while widely considered evil in SQL, is represented by a 16-bit unsigned int and is pretty fast -- as long as you never need to query what the values are (e.g., it's awful for dropdowns).

That said, I can imagine us adding more 'lists' somewhere down the line -- perhaps a gray-list, or a teal list. And 1 byte vs 11 bytes isn't a huge bloat.

I am in favor of keeping this as a char().